src/js/json.js

```diff
@@ -0,0 +1,24 @@
+// Module to add body parsing to express.
+"use strict";
+var fluid  = fluid || require("infusion");
+var gpii   = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.express.middleware.bodyparser.json");
+
+var bp         = require("body-parser");
+
+gpii.express.middleware.bodyparser.json.getMiddlewareFunction = function () {
+    return bp.json();
+};
+
+fluid.defaults("gpii.express.middleware.bodyparser.json", {
+    gradeNames: ["fluid.standardRelayComponent", "gpii.express.middleware", "autoInit"],
+    model: {
```

Comment:
The whole `model` property was cruft from a previous approach.  I removed the definition altogether and simplified the `gradeNames` as well.